### PR TITLE
Docs: Add example to extract a specific label value

### PR DESCRIPTION
**Pull request by simonc6372**
As a non-go programmer, this took me ages to figure out. Adding it in as an example should be helpful.

**What is this feature?**

A small docs improvement. 

**Why do we need this feature?**

As a non-go programmer, this took me ages to figure out. Adding it in as an example should be helpful.

**Who is this feature for?**

Docs change. 

**Which issue(s) does this PR fix?**:

n/a 

Fixes #

**Special notes for your reviewer**:
My first PR to Grafana. Please advise if I'm not doing this right.

### CODE REVIEW

File: docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md [hunk 1]
```diff
@@ -71,6 +71,14 @@ The name of the annotation is {{ .Name }}, and the value is {{ .Value }}
 {{ end }}
 ```
 
+## Use a specific label value
```
Comment (grobinson-grafana) [inline docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md:74]:
```suggestion
## The index function
```

File: docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md [hunk 2]
```diff
@@ -71,6 +71,14 @@ The name of the annotation is {{ .Name }}, and the value is {{ .Value }}
...
+
+You may only want to use some specific label values in your text, or use a longer description.
```
Comment (grobinson-grafana) [inline docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md:76]:
```suggestion
To print a specific annotation or label you can use the `index` function.
```

File: docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md [hunk 3]
```diff
@@ -71,6 +71,14 @@ The name of the annotation is {{ .Name }}, and the value is {{ .Value }}
 {{ end }}
 ```
 
+## Use a specific label value
+
+You may only want to use some specific label values in your text, or use a longer description. 
+
+```
+This alert is in the folder {{ index .Labels "grafana_folder" }}
```
Comment (grobinson-grafana) [inline docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md:79]: Would be great if we can put this in a range. A lot of users don't look at the context of code examples and instead just use them verbatim and then complain it doesn't work, so adding a `range` here should make it work if copied without reading the rest of the page.

```suggestion
{{ range .Alerts }}
This alert is in the folder {{ index .Labels "grafana_folder" }}
{{ end }}
```

**Review by grobinson-grafana — CHANGES_REQUESTED** [review]
Looks great! I requested some changes to make the example more generic as the index function has much wider use cases. Thank you for helping improve the docs! ❤️

File: docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md (hunk 2 above)
Comment (brendamuir) [reply docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md:76]: To print a specific annotation or label, use the 'index' function.

**Review by brendamuir — APPROVED** [review]

**Comment by brendamuir** [comment]
@grobinson-grafana are we ready to approve this one and merge?

**Review by grobinson-grafana — APPROVED** [review]

**Pull request merged**